### PR TITLE
adds happiness, resolves #224

### LINKED ISSUE #224: Add happiness
We need to add https://github.com/JedWatson/happiness to this repository.

Pull requests welcome.

### PR DESCRIPTION
**Pull request by dcousens**
Currently the tests are failing because eslint explodes.

### CODE REVIEW

**Comment by bruderstein** [comment]
You don't need to do `npm run coveralls` and `npm run unit`, `coveralls` runs the tests, and fails if the tests fail.

**Comment by dcousens** [comment]
@bruderstein perhaps then `coveralls` is doing too much?
I'd expect that target to run all the way even if the tests fail?
No?

**Comment by bruderstein** [comment]
To test coverage, we run the tests. That's what coverage is :smile:

If we make this pass even if the tests fail (just add an `|| true`), then the tests run twice, which is just extra work with no benefit.

**Comment by dcousens** [comment]
@bruderstein I've also found keeping coveralls separate helps when coveralls fails,  which with my experience in https://github.com/bitcoinjs/bitcoinjs-lib/,  happens often.

For comparison, our `scripts` can be found [here].
I agree though,  it does seem a bit redundant,  and I'm on the fence about whether or not to change it.  If only coveralls didn't explode so often.

File: package.json [hunk 1]
```diff
@@ -48,11 +49,12 @@
     "cover": "istanbul cover node_modules/.bin/_mocha -- -u exports --compilers js:babel/register -R spec",
     "coveralls": "NODE_ENV=test ./node_modules/.bin/istanbul cover ./node_modules/mocha/bin/_mocha --report lcovonly -- -u exports --compilers js:babel/register -R spec && cat ./coverage/lcov.info | ./node_modules/coveralls/bin/coveralls.js",
```
Comment (dcousens) [inline package.json]: @bruderstein why not just `| coveralls` by the way?

Comment (bruderstein) [inline package.json]: I took that from the coveralls site, if there's a better way, let's have it.

Comment (dcousens) [inline package.json]: @bruderstein changed in https://github.com/JedWatson/react-select/commit/65358b5f8a477d68f21bc83485d7975785c5b162

**Comment by dcousens** [comment]
I'll leave this one to you @bruderstein

**Pull request closed without merging**